### ISSUE TITLE
Error: sample CXL_FDSW202345319-1r not found in JSON library file.

### ISSUE THREAD

**Issue by ChenDepp** [brentp/smoove#141]
@brentp 
  hi , it report a error when i running smoove. the command is：smoove call -x --name CXL --fasta ../Ref/CXL-1.build.chr.fa --processes 10 --genotype CXL.sort.fixmate.rmdup.bam。
     the error is :
       
![QQ截图20210105142813](https://user-images.githubusercontent.com/48479509/103613834-43558700-4f62-11eb-8769-01d1fa4bc8c8.png)

Is this problem caused by the sample name containing Chinese ?（my sample name is 穿心莲_FDSW202345319-1r)

**Comment by brentp** [comment]
yes, the naming is almost certainly the problem. Unfortunately, you'll have to stick to simple ascii.

**Comment by ChenDepp** [comment]
@brentp  
  thanks for your reply. good luck
